presto-main/src/main/java/com/facebook/presto/sql/planner/plan/AggregationNode.java

```diff
@@ -48,6 +48,16 @@
     private final Optional<Symbol> hashSymbol;
     private final Optional<Symbol> groupIdSymbol;
 
+    public boolean hasEmptyGroupingSet()
```

Comment:
If these are only used in one file, make them helper functions in that file instead.